src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+```
+if the output is:
+```
+| status        | uploaded                           |
+```
+then it was successfully uploaded. Otherwise, wait for it to finish uploading or wait check common troubleshooting commands.
+
+Now, let's apply the app:
+```
+[sysadmin@controller-0 ~(keystone_admin)]$ system application-apply poc-starlingx-app
+``` 
+
+If you never installed a StarlingX app before, know that it is normal to take some minutes before finishing the applying command. We can check that the application was successfuly applied:
+
+```
+[sysadmin@controller-0 ~(keystone_admin)]$ system application-show poc-starlingx-app | grep applied
+| status        | applied
+```
+
+If you followed me until here, and your application was successfully applied, then we're done! In case you run into any problems, please check the troubleshooting section.
+
+## Troubleshooting
+### sysinv log
+
+Usually, after seeing the application has failed, this is problably the best first place for you to check out and find out more about what has gone wrong.
+The sysinv log is located on:
+```
+/var/log/sysinv.log
+```
+It may be useful using:
+```
+cat /var/log/sysinv.log | grep "application name"
+```
+
+### system application commands
+these commands may be beneficial for finding out about the state of an application when it did not deploy correctly and you want to reinstall or manage them. Some commands are:  
+* ```system application-list ```
+	- this command shows the list of applications uploaded or applied to the cluster and some top-level informations. Particularly, status and progress are useful for debbuging.
+* ```system application-show "application_name"```
+	- this command provides a most of the information provided in the _application-list_, but with extra rows: ["Active", "created_at", "name", "updated_at"].
+* ```system application-remove```
+	- this command removes an application from service. Removing an application will clean up related Kubernetes resources and delete all of its installed helm charts
+* ```system application-delete```
+	- we can use the following command to completely delete an application from the system **after running the system application-remove**
+
+more useful [system application commands](https://docs.starlingx.io/system_configuration/kubernetes/application-commands-and-helm-overrides.html)
+
+[bug when trying to remove and delete an application](https://wiki.openstack.org/wiki/StarlingX/Containers/FAQ#What_to_I_do_if_I_can.27t_re-run_an_application_action.3F)
+### kubernetes pod logs
+
+In some cases, You'll in the sysinv.log that the application of a pod got created but for some reason It is not working properly. In those cases It is useful checking the kubernetes log of that pod. Basic command is:
+```
+kubectl logs podname -n namespace
+```
+also note that the infos like the prefix of the pod and the namespace where It will be created and running are written in the helm chart of the application.
```

Comment:
Ildiko asked bruno and me to add this section in the end, I think you can go ahead and do the same

# About StarlingX

If you would like to learn more about the project and get involved check the [website](https://www.starlingx.io) for more information or [download the code](https://opendev.org/starlingx) and start to experiment with the platform. If you are already evaluating or using the software please fill out the [user survey](https://openinfrafoundation.formstack.com/forms/starlingx_user_survey) and help the community improve the project based on your feedback.

Reply:
ack